src/pages/project/helpers/projectContributorHelpers.ts

```diff
@@ -11,3 +13,80 @@ export const getNonProjectManagerContributors = (contributors: ProjectContributo
 
 export const hasUnidentifiedContributor = (contributors: ProjectContributor[]) =>
   contributors.some((contributor) => !contributor.identity || !contributor.identity.id);
+
+export enum AddContributorErrors {
+  NO_PERSON_TO_ADD = 'NO_PERSON_TO_ADD',
+  SAME_ROLE_WITH_SAME_AFFILIATION = 'SAME_ROLE_WITH_AFFILIATION',
+  ALREADY_HAS_A_PROJECT_MANAGER = 'ALREADY_HAS_A_PROJECT_MANAGER',
+}
+
+export const addContributor = (
+  personToAdd: CristinPerson | undefined,
+  contributors: ProjectContributor[],
+  roleToAddTo: ProjectContributorType
+): { newContributors?: ProjectContributor[]; error?: string } => {
```

Comment:
Om disse ikke må matche noe bestemt en annen plass kan man like godt bare bruke default-verdi fra enum også. Slipper feks å tenke på at key/value må matche

```suggestion
  NO_PERSON_TO_ADD,
  SAME_ROLE_WITH_SAME_AFFILIATION,
  ALREADY_HAS_A_PROJECT_MANAGER,
}

export const addContributor = (
  personToAdd: CristinPerson | undefined,
  contributors: ProjectContributor[],
  roleToAddTo: ProjectContributorType
): { newContributors?: ProjectContributor[]; error?: AddContributorErrors } => {
```